### PR TITLE
Port of #5667: Corrects various misspellings, grammar, & consistency issues 

### PR DESCRIPTION
**Pull request by EmilitiaEnnehrt**
## About The Pull Request
It says 87 files changed, that is because I caught a few extra mistakes which are put below in comments.

[Original PR 5667](https://github.com/sojourn-13/sojourn-station/pull/5667/files). Made very small changes and equally did not apply a balance change that was sneaked in there.
> Check the original PR's explanation of some changes there, if you disagree or want changes, please feel free to comment.

Not to be confused with [my closed PR](https://github.com/Equinox-SS13/equinox-sojourn/pull/81), it will take its own come back soon.

	
<hr>
</details>

## Changelog
:cl:
fix:corrects various misspellings, grammar, & consistency issues
/:cl:

### CODE REVIEW

**Comment by EmilitiaEnnehrt** [comment]
Okay, I've gone through and verified every single line. I can vouch for all of them being identical to the original PR port with the exception of `line 219` in `/task_master/tasks` (which adjusts level for sheath arts, if you see fit I can add it here or make it a separaate PR).

The one I have no idea what's going on is `CONTRIBUTING.md` and would appreciate some guidance.

**Review by Michiyamenotehifunana — APPROVED** [review]
Everything looks good to me

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,30 +1,32 @@
...
+-   Pull requests should not contain changes that do not relate with functionality described in commit messages.
+-   If pull request relates with existing github issue, it should be specified in commit message, for example, "Fix broken floor sprites, close #23" (see https://help.github.com/articles/closing-issues-via-commit-messages/ for additional info).
+-   If pull request contains map files changes, it MUST be fitted for the correct format. External tools like StrongDMM automatically do this. Local in-code tools currently do not work so it is best to use proper tools than Dream Makers base. Pull request description should contain screenshots of map changes if it's not obvious from map files diff.
+-   If pull request contains icon files changes, the pull request description should contain screenshots of changed icon files. While this isn't required, it's more preferred than digging into the file or checking the Actions Tools for your sprite changes.
+-   If you have the permissions, please set appropriate labels on your PRs. Including, at the very least, Good-To-Merge to indicate that its done.
```
[the remark this answers is not in the document]
Comment (valkyria-gk) [reply CONTRIBUTING.md:10]: It is just how MD is supposed to work, I guess.

File: code/modules/mob/living/carbon/superior_animal/psi_monsters/types/Ploge.dm [hunk 2]
```diff
@@ -1,4 +1,4 @@
-//EXSTREAMLY RARE able heck some people up or do events with
+//EXTREMELY RARE able heck some people up or do events with
```
[the remark this answers is not in the document]
Comment (valkyria-gk) [reply code/modules/mob/living/carbon/superior_animal/psi_monsters/types/Ploge.dm:1]: My guess it is a family-friendly way to say 'fuck', being an exclamation.

File: modular_sojourn/bottler.dm [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /obj/item/bottle_kit
-	name = "Bottling Kit"
-	desc = "A box that holds glasses fabricators, lables, caps and quarks ready for home brewing and bottling."
+	name = "bottling Kit"
```
Comment (valkyria-gk) [inline modular_sojourn/bottler.dm:2]: This is a weird change. Did the OP say why they changed the capitalisation?

File: modular_sojourn/brewing.dm [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /obj/structure/fermentation_keg
-	name = "Fermentation Keg"
+	name = "fermentation Keg"
```
Comment (valkyria-gk) [inline modular_sojourn/brewing.dm:2]: Same here.

File: modular_sojourn/spaceleak.dm [hunk 5]
```diff
@@ -1,6 +1,6 @@
 
 /obj/item/bluespace_leak
-	name = "Leaking bluespace"
+	name = "leaking bluespace"
```
Comment (valkyria-gk) [inline modular_sojourn/spaceleak.dm:3]: Same with capitalization here. The wording doesn't roll off good either, it is also not what we usually would say like:
- Spacetime Rift
- Rift
- Portal
- Bluespace Leak
- Blue(space) Rift

File: modular_sojourn/bottler.dm (hunk 3 above)
Comment (EmilitiaEnnehrt) [reply modular_sojourn/bottler.dm:2]: Actually I think it's just me forgetting to decapitalize Kit to kit, thanks for this oversight.

File: modular_sojourn/spaceleak.dm (hunk 5 above)
Comment (EmilitiaEnnehrt) [reply modular_sojourn/spaceleak.dm:3]: Will do this, was thinking back to the comment on decapitalized Bluespace Artillery on my closed PR, personally I think two capitalized is preferable.

File: code/modules/mob/living/carbon/superior_animal/psi_monsters/types/Ploge.dm (hunk 2 above)
[the remark this answers is not in the document]
Comment (EmilitiaEnnehrt) [reply code/modules/mob/living/carbon/superior_animal/psi_monsters/types/Ploge.dm:1]: Want me to change to mess or is this not needed?

[the remark this answers is not in the document]
Comment (valkyria-gk) [reply code/modules/mob/living/carbon/superior_animal/psi_monsters/types/Ploge.dm:1]: No need. :)

**Review by valkyria-gk — APPROVED** [review]

**Pull request merged**